### PR TITLE
Release v1.0

### PR DESCRIPTION
**Pull request by AlexForJava**

### CODE REVIEW

File: src/com/gmail/chernii/oleksii/Main.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            int second = Integer.valueOf(sc.nextLine());
+            System.out.println("Enter action");
+            char action = sc.nextLine().charAt(0);
+            System.out.println("answer = " + calculator.calculate(first, second, action));
+            System.out.println("Stop? - Yes/No");
+            stop = (sc.nextLine().equalsIgnoreCase("yes")) ? true : false;
+        }
+    }
+
+    private static int intIllegalArgumentException() {
```
Comment (Taniann) [inline src/com/gmail/chernii/oleksii/Main.java:34]: better make custom exception as separate class

File: src/com/gmail/chernii/oleksii/Main.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+/**
+ * Created by Space on 07.03.2019.
+ */
+public class Main {
+    private static Calculator calculator = (first, second, action) ->
+            (action == '+') ? (first + second) :
+                    (action == '*') ? (first * second) :
+                            (action == '-') ? (first - second) :
+                                    (action == '$' && second != 0) ? (Math.pow(first, 1. / second)) :
```
Comment (Taniann) [inline src/com/gmail/chernii/oleksii/Main.java]: where is dividing "/" ?

**Review by Taniann — CHANGES_REQUESTED** [review]
read comments

**Comment by AlexForJava** [comment]
Custom exeption insert into ternary operator? 

10 марта 2019, 16:37:16, от "Алексей" <hqq@ukr.net>:

@Taniann requested changes on this pull request.

read comments

In src/com/gmail/chernii/oleksii/Main.java:

> +        Scanner sc = new Scanner(System.in);
+        boolean stop = false;
+        while (!stop) {
+            System.out.println("Enter first number");
+            int first = Integer.valueOf(sc.nextLine());
+            System.out.println("Enter second number");
+            int second = Integer.valueOf(sc.nextLine());
+            System.out.println("Enter action");
+            char action = sc.nextLine().charAt(0);
+            System.out.println("answer = " + calculator.calculate(first, second, action));
+            System.out.println("Stop? - Yes/No");
+            stop = (sc.nextLine().equalsIgnoreCase("yes")) ? true : false;
+        }
+    }
+
+    private static int intIllegalArgumentException() {
better make custom exception as separate class

In src/com/gmail/chernii/oleksii/Main.java:

> @@ -0,0 +1,36 @@
+package com.gmail.chernii.oleksii;
+
+import java.util.Scanner;
+
+/**
+ * Created by Space on 07.03.2019.
+ */
+public class Main {
+    private static Calculator calculator = (first, second, action) ->
+            (action == '+') ? (first + second) :
+                    (action == '*') ? (first * second) :
+                            (action == '-') ? (first - second) :
+                                    (action == '$' && second != 0) ? (Math.pow(first, 1. / second)) :
where is dividing "/" ?

**Comment by Taniann** [comment]
Yes, you can insert it in ternary operator. Or change your ternary operators with map

**Comment by Taniann** [comment]
No, sorry
You can not throw exception in ternary operator
Better change ternary operator with map

**Pull request closed without merging**